### ISSUE TITLE
Emperor request and military not displayed in the interface + monumental temple unavailable

### ISSUE THREAD

**Issue by Alexandriusktarius** [Keriew/augustus#910]
Hello,
After installing Augustus 3.2 the inteface does not display the emperor request nor the military.
Monumental Temple + Coliseum + Hippodrome are not available.
How can I fix that.
Thank you
![1](https://github.com/Keriew/augustus/assets/138197550/eb67f352-3fdb-48e7-8435-0aeacc43876d)

**Comment by crudelios** [comment]
Regarding the emperor requests and objectives, you need to have enough height in the resolution for it to show, which doesn't seem to be the case.

If you're already playing on full screen you can reduce the display scaling so that extra info fits, but it will make everything smaller.

For the military sidebar you need to enable it in the settings and then click a military unit.

Regarding the lack of monuments, you may be playing on a city which doesn't allow you to build them. Which city are you playing?

**Comment by Alexandriusktarius** [comment]
Thank you for reply! Regarding monuments I updated the game and they work now. See in the attached screenshot ( first one is a saved game with no monuments, second is a restart on the same map). So I guess my ceasar 3 game was on version 1.0 and that caused the issue. 
I scaled the display as you said and the emperor request and objectves are displayed now. Thanks
![1](https://github.com/Keriew/augustus/assets/138197550/cdd60bc3-847e-40aa-bce0-e5750f7fcde9)
![2](https://github.com/Keriew/augustus/assets/138197550/5e440b08-7697-47af-b545-667b5fae01fb)
